backend/lib/migrations/simulations/to-v15.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Extrait une question service_civique à partir
+ * de la question activite
+ */
+
+const VERSION = 15
+
+function getAge(simulation) {
+  let age = simulation.answers.all.find(
+    (answer) => answer.id === "demandeur" && answer.fieldName === "age"
+  )?.value
+
+  if (age !== undefined) {
+    return age
+  }
+
+  const dateOBirth = simulation.answers.all.find(
+    (answer) =>
+      answer.id === "demandeur" && answer.fieldName === "date_naissance"
+  )?.value
+
+  if (!dateOBirth) {
+    // Dans ce cas on inserera la question service_civique
+    // car undefined > 31 est false
+    return
+  }
+
+  const dt = new Date(simulation.dateDeValeur)
+  const dob = new Date(dateOBirth)
+  age = Math.round(
+    (dt.getTime() - dob.getTime()) / 365.25 / 24 / 60 / 60 / 1000
+  )
```

Comment:
Je suis étonné que ce code ne soit pas mutualisé quelque part, il y avait un peu plus court pour le faire :
```suggestion
age = dayjs(simulation.dateDeValeur).diff(dateOBirth, "year")
```

Reply:
j'ai changé pour ta proposition qui est plus concise, mais pas lié à la méthode d'individu :-)